package.json

```diff
@@ -4,18 +4,22 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "babel-app": "babel app.js --out-file dist/app.js",
-    "babel-server": "babel server/ -d dist/server/ && npm run babel-app",
-    "build": "npm run clean && mkdir dist/ && npm run babel-server",
+    "start": "npm run build && node dist/app.js",
     "clean": "rm -rf dist/",
-    "test": "mocha --require babel-register server/tests/*.test.js",
+    "build": "npm run clean && mkdir dist/ && babel server/ -d dist/",
```

Comment:
at what point are you going to run the pretest and posttests scripts?

Reply:
so if i want to run the tests locally i'd have to run `npm run pretest &&  npm run test  && npm run posttest? `

Reply:
@seyi-adeleke not at al. By default calling npm test runs the pretest script first and the posttest after the test is run.

Reply:
Nice 👍